### PR TITLE
Another attempt at alignments

### LINKED ISSUE #4456: Refactor Blockbase Alignments
Due to recent changes in Gutenberg that brought the 'container-' class to all containers this can no longer be leveraged to help control Blockbase alignments.

How blockbase is configuring and managing alignment state needs to be revisited.

# Related:
#4435 (noted alignment + Global Styles clobbering issues with Cover and Group blocks)
#4448 (a potential fix to the issue that was causing pain in Skatepark)
#4454 (Removes alignment styling and gets out of the alignment ponyfill game)

### PR DESCRIPTION
**Pull request by pbking**
WIP...

### CODE REVIEW

File: blockbase/sass/base/_alignment.scss [hunk 1]
```diff
@@ -1,17 +1,36 @@
-.wp-block-group.alignfull,
-*[class*="wp-container-"] //Anything that inherits layout (container)
-{
-	//give it some padding
-	padding-left: var(--wp--custom--post-content--padding--left);
-	padding-right: var(--wp--custom--post-content--padding--right);
+//FRONTEND
+.blockbase-alignment-rules, //for argeted elements
```
Comment (scruffian) [inline blockbase/sass/base/_alignment.scss]:
```suggestion
.blockbase-alignment-rules, //for targeted elements
```

File: blockbase/sass/base/_alignment.scss [hunk 2]
```diff
@@ -1,17 +1,36 @@
...
+	}
+}
 
-	//Any nested containers, and anything that is alignfull
-	*[class*="wp-container-"], // Any nested containers
-	> .alignfull { // Any direct descendant that is alignfull
-		// bust out of the container's padding
+// EDITOR (NOTE: It PROBABLY would be OK to bring these together to "simplify" the stylesheet.  However the selectors are quite different
+// and it's a lot easier to understand and ensure intent separated in this way.
+.is-root-container { //top level of the editor
```
Comment (scruffian) [inline blockbase/sass/base/_alignment.scss:23]: does this also work in the site editor?

**Comment by scruffian** [comment]
It would be useful to document the different cases this is trying to address...

File: blockbase/sass/base/_alignment.scss (hunk 2 above)
Comment (pbking) [reply blockbase/sass/base/_alignment.scss:23]: Yes.  We looked carefully at the editor.  It seems to cover the same use-cases as the view.

**Comment by pbking** [comment]
They are documented here: pcrugM-lB-p2

File: quadrat/block-templates/index.html [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <!-- wp:template-part {"slug":"header"} /-->
 
-<!-- wp:query {"tagName":"main","layout":{"inherit":true},"queryId":1,"query":{"perPage":5,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","sticky":"","inherit":true}} -->
+<!-- wp:query {"tagName":"main",layout":{"inherit":true},"queryId":1,"query":{"perPage":5,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","sticky":"","inherit":true}} -->
```
Comment (MaggieCabrera) [inline quadrat/block-templates/index.html]: I think we are missing an " on layout" here and in some of the other templates

File: blockbase/sass/base/_alignment.scss [hunk 4]
```diff
@@ -1,17 +1,36 @@
...
-*[class*="wp-container-"] //Anything that inherits layout (container)
-{
-	//give it some padding
-	padding-left: var(--wp--custom--post-content--padding--left);
-	padding-right: var(--wp--custom--post-content--padding--right);
+//FRONTEND
+.wp-site-blocks { // top level of the view
+	//In this situation we want to introduce a standardized gap between content and the edge of the screen.
+	margin-left: var(--wp--custom--post-content--padding--left);
+	margin-right: var(--wp--custom--post-content--padding--right);
```
Comment (MaggieCabrera) [inline blockbase/sass/base/_alignment.scss]: I think this can be padding, I tested and nothing seemed to break

**Review by MaggieCabrera — APPROVED** [review]
🚢

**Pull request merged**